package.json

```diff
@@ -23,7 +23,7 @@
     "test": "jest --env=node --forceExit",
     "build": "babel --presets @babel/preset-react -d lib/ src && tsc",
     "tsc": "tsc",
-    "test_integration": "npm run test_network_up && npm run test_network_integration && npm run test_network_down || npm run test_network_partial_down",
+    "test_integration": "npm run test_network_up && npm run test_network_integration && npm run test_network_down",
```

Comment:
This was a command that was run when there were errors in the test, to deactivate only the `cpuminer` instead of destroying the whole `privatenet`.

Trouble is: it always returned a success for the `cpuminer` shutdown. This was interpreted by GitHub's CI as if the tests succeeded.
So I had to remove it. More info on the PR's description.